lib/administrate/base_dashboard.rb

```diff
@@ -8,6 +8,7 @@
 require "administrate/fields/number"
 require "administrate/fields/polymorphic"
 require "administrate/fields/string"
+require "administrate/fields/password"
```

Comment:
When a list like this doesn't have any inherent order, thoughtbot prefers to sort them alphabetically. Would you mind doing that here?